ci/github/build_driver_container

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+build_driver_container () {
+      cd ci/kmm-kmod && make KERNEL_SRC_DIR="/usr/src/linux-headers-$(uname -r)" && \
+      docker build -t kmm-kmod:local --build-arg=KERNEL_VERSION=$(uname -r) . && \
+      minikube image load kmm-kmod:local 
+}
+
+build_driver_container
+
```

Comment:
IMO, we should have one file to contain everything needed in order for preparing the infra.
Having function in it is great but no need to split it into multiple files.

Also, this should be part of the infra step as the m/s CI will build all the images for us.